api/task_delete.go

```diff
@@ -7,7 +7,7 @@ import (
 	"github.com/hashicorp/consul-terraform-sync/logging"
 )
 
-// DeleteTaskByName deletes an existing task and its events. Does not delete
+// DeleteTaskByName deletes an existing task and its events asynchronously. Does not delete
 // if the task is active.
```

Comment:
```suggestion
// until the task is inactive and not running.
```
This line is a little out of date, I forgot to update it when it was changed from error if active to waiting until it was inactive.